index.html

```diff
@@ -31,7 +31,7 @@
                     </li>
                     <!-- wishlist and cart icons -->
                     <li class="nav-icon position-r">
-                        <a href="#">
+                        <a href="./pages/wishlist/wishlist.html">
                             <img class="img-100 cursor-pr"
                                 src="https://ui-phoenix.netlify.app/Assets/svg/navigation/heart-hollow.svg"
```

Comment:
The code looks nice. Just one suggestion, it would be good if you will use `loading=lazy` with `<img />`. As it will improve the performance of your app.

Reply:
Sure ! will do that.